packages/mail/src/mail.spec.js

```diff
@@ -53,19 +53,84 @@ describe('sgMail.send()', () => {
     }).to.throw(Error);
   });
 
-  it('should include custom headers to the request', () => {
-    sgClient.setDefaultHeader('X-Mock', 201);
+  it('should include custom headers to the request', async () => {
+    sgClient.setDefaultHeader('X-Mock', 202);
     const clientSpy = sinon.spy(sgClient, "request")
     return sgMail
       .send(Object.assign(data, { headers: { customHeader: "Custom Header Content" } }))
       .then(([response, body]) => {
-        expect(response.statusCode).to.equal(201);
+        expect(response.statusCode).to.equal(202);
         expect(clientSpy).to.have.been.calledWith(sinon.match({
           url: "/v3/mail/send",
           method: "POST",
           headers: { customHeader: "Custom Header Content" }
         }));
       });
   });
+
+  it('should send email with correct replyToList format', async () => {
```

Comment:
Can you also add an example [here](https://github.com/sendgrid/sendgrid-nodejs/blob/main/docs/use-cases/cc-bcc-reply-to.md)?